text/0006-architecture-review-process.md

```diff
@@ -11,27 +11,29 @@ layout: text
+The function of a **Roadmap Review** is to decide if a thing should be done. The goal is to bring together a packet of data to inform a management decision to provide resources to make the thing happen. A Roadmap Review should happen early in the process so that build time isn’t wasted on a “No” decision, but so that enough information is available to management.
 
 A **Design Review** is to decide if the thing should be done in a particular way. The goal is to ensure that the problem space is understood well enough that a realistic bug breakdown can be made, that those working on the problem all agree on the challenge, and that work can be prioritized to hit the hardest parts of the problem first (in order to get early warning if the hard parts prove impossible or to inform schedule projections early).
 
 ## Timetable
 
 For both a Roadmap Review and a Design Review the **timetable** is as follows:
 
-1. The team should set the stage (see "initial meeting" below):
-  * Summarize in a one or two sentences the proposal to be reviewed. This should be agreed between the team asking for review and the chair.
-  * Find someone to **chair** the review:
+1. The team should set the stage:
+  * Summarize in a one or two sentences the proposal to be reviewed. This should be agreed between the team asking for review and the reviewer.
+  * Find a **reviewer**:
     - For a Roadmap Review this will be the least senior manager with the ability to address the problem in its entirety.
     - For a Design Review this will be an experienced engineer in the problem domain outside of the team willing to put time into the problem. As engineers don’t review their own code they shouldn’t review their own designs. The principle is to get the most informed and least biased feedback possible.
+  * Optionally, for larger reviews, find someone to **chair** the review. This enables to reviewer to concentrate on the problem rather than the admin of the review.
 
 2. The team produces a **Review Packet** designed to document the proposal identified in step 1. The production of the Review Packet is expected to be collaborative and iterative to ensure that the review focuses on the key issues so the team presents its best case. This is particularly true of a Roadmap Review where the consequences of the review are likely to be more significant.
 The Review Packet includes:
-  * A lay summary of the problem space and the stakeholders which is focused on defining a shared language and identifying the key forces behind the problem
+  * A lay summary of the problem space which is focused on defining a shared language and identifying the key forces behind the problem
```

Comment:
We did include a 'stakeholders' section for the XBL review, and I think having that was valuable. But maybe it should be pulled out into a separate bullet

Reply:
OK. The logic for me removing that was partly the clumsy language, but also that the stakeholders  should know who they are because they are at the review meeting. However on reflection, documenting this does make the thought process more obvious.

I'll add as a separate bullet that the review packet includes "_A list of the groups directly affected by the proposal to ensure that the review meeting includes representatives from those groups._"

See 3d318b3e1850c20794d174fb727180ec42c7aad0.